### PR TITLE
Update dependency and plugin versions

### PR DESCRIPTION
**Pull request by brfrn169**
This PR updates the dependency and plugin versions. Please take a look!

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -318,7 +318,7 @@ jobs:
       - name: Setup and execute Gradle 'integrationTestJdbc' task
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:sqlserver://localhost:1433 -Dscalardb.jdbc.username=sa -Dscalardb.jdbc.password=SqlServer19
+          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:sqlserver://localhost:1433;encrypt=true;trustServerCertificate=true -Dscalardb.jdbc.username=sa -Dscalardb.jdbc.password=SqlServer19
```
Comment (brfrn169) [inline .github/workflows/ci.yaml:321]: For the new SQL Server Driver version, we need these parameters according to the following article:
https://stackoverflow.com/questions/73291498/sql-server-spring-boot-pkix-path-building-failed-unable-to-find-valid-certific

File: schema-loader/src/test/java/com/scalar/db/schemaloader/command/CassandraCommandTest.java [hunk 2]
```diff
@@ -336,6 +336,6 @@ public void call_MissingSchemaFile_ShouldExitWithErrorCode() {
     // Assert
     Assertions.assertThat(exitCode).isEqualTo(ExitCode.USAGE);
     Assertions.assertThat(stringWriter.toString())
-        .contains("Missing required option '--schema-file=<schemaFile>'");
+        .contains("Missing required option: '--schema-file=<schemaFile>'");
```
Comment (brfrn169) [inline schema-loader/src/test/java/com/scalar/db/schemaloader/command/CassandraCommandTest.java:339]: In the new picocli version, the error message is slightly changed.

File: schema-loader/src/test/java/com/scalar/db/schemaloader/command/CosmosCommandTest.java [hunk 3]
```diff
@@ -224,6 +224,6 @@ public void call_MissingSchemaFile_ShouldExitWithErrorCode() {
     // Assert
     Assertions.assertThat(exitCode).isEqualTo(ExitCode.USAGE);
     Assertions.assertThat(stringWriter.toString())
-        .contains("Missing required option '--schema-file=<schemaFile>'");
+        .contains("Missing required option: '--schema-file=<schemaFile>'");
```
Comment (brfrn169) [inline schema-loader/src/test/java/com/scalar/db/schemaloader/command/CosmosCommandTest.java:227]: Ditto.

File: schema-loader/src/test/java/com/scalar/db/schemaloader/command/DynamoCommandTest.java [hunk 4]
```diff
@@ -360,6 +360,6 @@ public void call_MissingSchemaFile_ShouldExitWithErrorCode() {
     // Assert
     Assertions.assertThat(exitCode).isEqualTo(ExitCode.USAGE);
     Assertions.assertThat(stringWriter.toString())
-        .contains("Missing required option '--schema-file=<schemaFile>'");
+        .contains("Missing required option: '--schema-file=<schemaFile>'");
```
Comment (brfrn169) [inline schema-loader/src/test/java/com/scalar/db/schemaloader/command/DynamoCommandTest.java:363]: Ditto.

File: schema-loader/src/test/java/com/scalar/db/schemaloader/command/JdbcCommandTest.java [hunk 5]
```diff
@@ -220,6 +220,6 @@ public void call_MissingSchemaFile_ShouldExitWithErrorCode() {
     // Assert
     Assertions.assertThat(exitCode).isEqualTo(ExitCode.USAGE);
     Assertions.assertThat(stringWriter.toString())
-        .contains("Missing required option '--schema-file=<schemaFile>'");
+        .contains("Missing required option: '--schema-file=<schemaFile>'");
```
Comment (brfrn169) [inline schema-loader/src/test/java/com/scalar/db/schemaloader/command/JdbcCommandTest.java:223]: Ditto.

File: server/src/main/java/com/scalar/db/server/AdminService.java [hunk 6]
```diff
@@ -64,13 +63,6 @@ public void unpause(Empty request, StreamObserver<Empty> responseObserver) {
     responseObserver.onCompleted();
   }
 
-  @Override
-  public void stats(Empty request, StreamObserver<StatsResponse> responseObserver) {
-    // returns empty for now
-    responseObserver.onNext(StatsResponse.newBuilder().setStats("{}").build());
-    responseObserver.onCompleted();
-  }
-
```
Comment (brfrn169) [inline server/src/main/java/com/scalar/db/server/AdminService.java:73]: In the new Scalar Admin version, this `stats` method is deleted..

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by komamitsu — APPROVED** [review]
LGTM, thank you!

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**